tests/test_app.py

```diff
@@ -1,21 +1,20 @@
-#!/usr/bin/python
-# -*- coding: utf-8 -*-
+#!/usr/bin/env python
 
-from flask import json
 import pytest
+from flask import json
 
 
 @pytest.mark.parametrize('name, status',
-    [
-        ('Shin', 200),
-        ('Tomoya', 200),
-        ('Masato', 200),
-    ])
+                         [
+                             ('Shin', 200),
+                             ('Tomoya', 200),
+                             ('Masato', 200),
+                         ])
 def test_app_server_username(client, name, status):
-    response = client.get('/user/'+name)
+    response = client.get('/user/' + name)
     assert response.status_code == status
     data = json.loads(response.get_data(as_text=True))
-    assert len(data) > 0
+    assert 0 < len(data)
```

Comment:
細かいけれど、不等号は `<` `<=` の向きで使うことが推奨される。
なぜなら、数直線を左が小/右が大で書いたり、古来中国でも左より右が良い（**左遷**の語源）、日本語でも「右に出る者はいない」など、右が大きい方がわかりやすいため。

また、Pythonの場合、数値型の0は `False` 、それ以外は `True` のため、 `0 <` しなくても良い。
```suggestion
    assert len(data)
```